src/diffusers/image_processor.py

```
@@ -52,8 +52,16 @@ def __init__(
         resample: str = "lanczos",
         do_normalize: bool = True,
         do_convert_rgb: bool = False,
+        do_convert_grayscale: bool = False,
     ):
         super().__init__()
+        if do_convert_rgb and do_convert_grayscale:
+            warnings.warn(
```

maybe better to throw an error here actually